bundles/org.openhab.binding.evcc/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -295,7 +314,7 @@
 	<channel-type id="vehicleName">
 		<item-type>String</item-type>
 		<label>Vehicle Name</label>
-		<description>Name of EV</description>
+		<description>Name of EV (former vehicle-id)</description>
```

Comment:
When was this `vehicle-id`? And I'd prefer to only mention this in the README.

Reply:
https://github.com/evcc-io/evcc/releases/tag/0.123.0

> Vehicles are no longer addressed by index, but using their config `name`. `name` MUST NOT contain spaces or special characters.

Reply:
This was the major API change from 0.123.0. In the past the vehicles have been indexed.

Now:
![image](https://github.com/openhab/openhab-addons/assets/44850211/bad01b76-6495-406e-a9b5-a5827858b81c)

Reply:
Ah okay, thanks for the info.